lint-checks/src/test/java/com/googe/maps/android/lint/checks/GoogleMapDetectorTest.kt

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googe.maps.android.lint.checks
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+import com.google.maps.android.lint.checks.GoogleMapDetector
+
+@Suppress("UnstableApiUsage")
+class GoogleMapDetectorTest : LintDetectorTest() {
+
+    fun testSetOnMarkerDragListener() {
+        lint().files(
+            testFile(
+                """
```

Comment:
Nit - indentation for starting and ending quotes in this and other blocks are different - not sure if this is intended.

Reply:
Fixed